vaults/VAULT.md

```diff
@@ -0,0 +1,21 @@
+# Vault
+
+This folder contains multiple folders, each folder represents a chain, name of the folder is the chain id (e.g. `1` for Ethereum mainnet).
+
+Each folder contains JSON files of review vaults. Name of the file is the vault address. After the vault is reviewed, and deployed contract verified by SAM team, the vault file is added to specific folder depending on the network.
+
+JSON structure:
+
+- `riskLevel` is the final risk of the vault displayed on Yearn frontend.
+- `riskScore` is object with detailed values of each risk category.
+
+This data is used by [yDaemon](https://github.com/yearn/ydaemon). yDaemon fetches the vaults from blockchain registry and creates new JSON item for each vault. yDaemon was extended in [this commit](https://github.com/yearn/ydaemon/commit/b8296457af78cf97f41ef15cb502ff0744fd0a8b) to fetch the vault risk score from this folder.
```

Comment:
This section needs changes. The data from this repo is not going to be used yDaemon but by yCMS instead? It would be nice to reference the code that will use this data, so any changes in the structure can be easily referenced and updated if needed.